src/desktop/DesktopUtils.ts

```diff
@@ -201,13 +201,15 @@ export class DesktopUtils {
 	}
 
 	/**
-	 * Writes contents with a random file name into the directory of the executable
+	 * Writes contents with a random file name into the tmp directory
 	 * @param contents
 	 * @returns path to the written file
 	 */
 	private async _writeToDisk(contents: string): Promise<string> {
 		const filename = uint8ArrayToHex(this._desktopCrypto.randomBytes(12))
-		const filePath = swapFilename(process.execPath, filename)
+		const tmpPath = this.getTutanotaTempPath("reg")
```

Comment:
is temp dir private to us? Can someone swap file contents before we execute it?

Reply:
that was the thought behind putting it into the executables directory, since if you can mess with a file in there you can also just change the executable.

The tmp directory is user-private and can't be read by others without elevation.

Reply:
I rather meant other programs but I don't think it's an issue since we don't elevate for it anymore...